lib/node_modules/@stdlib/repl/lib/complete_expression.js

```diff
@@ -100,9 +100,7 @@ function complete( out, context, expression ) {
 		}
 		filter = node.expression.name;
 		debug( 'Identifier auto-completion. Filter: %s', filter );
-		out = filterByPrefix( out, RESERVED_KEYWORDS_COMMON, filter );
-		out = filterByPrefix( out, objectKeys( context ), filter );
-		out = filterByPrefix( out, ast.locals, filter );
+		out = filterByPrefix( out, RESERVED_KEYWORDS_COMMON.concat(objectKeys(context), ast.locals), filter ); // eslint-disable-line max-len
```

Comment:
Three separate invocations was intentional, as it avoids an additional unnecessary memory allocation. That was the point of providing an `out` argument in the first place. This change should be reverted.